x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/rule_event_log_list.tsx

```diff
@@ -0,0 +1,431 @@
+      'xpack.triggersActionsUI.sections.ruleDetails.eventLogColumn.newAlerts',
+      {
+        defaultMessage: 'New alerts',
+      }
+    ),
+    isSortable: SORTABLE_COLUMNS.includes('num_new_alerts'),
+  },
+  {
+    id: 'num_recovered_alerts',
+    displayAsText: i18n.translate(
+      'xpack.triggersActionsUI.sections.ruleDetails.eventLogColumn.recoveredAlerts',
+      {
+        defaultMessage: 'Recovered alerts',
+      }
+    ),
+    isSortable: SORTABLE_COLUMNS.includes('num_recovered_alerts'),
+  },
+  {
+    id: 'num_triggered_actions',
+    displayAsText: i18n.translate(
+      'xpack.triggersActionsUI.sections.ruleDetails.eventLogColumn.triggeredActions',
+      {
+        defaultMessage: 'Triggered actions',
+      }
+    ),
+    isSortable: SORTABLE_COLUMNS.includes('num_triggered_actions'),
+  },
+  {
+    id: 'num_succeeded_actions',
+    displayAsText: i18n.translate(
+      'xpack.triggersActionsUI.sections.ruleDetails.eventLogColumn.succeededActions',
+      {
+        defaultMessage: 'Succeeded actions',
+      }
+    ),
+    isSortable: SORTABLE_COLUMNS.includes('num_succeeded_actions'),
+  },
+  {
+    id: 'num_errored_actions',
+    displayAsText: i18n.translate(
+      'xpack.triggersActionsUI.sections.ruleDetails.eventLogColumn.erroredActions',
+      {
+        defaultMessage: 'Errored actions',
+      }
+    ),
+    isSortable: SORTABLE_COLUMNS.includes('num_errored_actions'),
+  },
+  {
+    id: 'total_search_duration',
+    displayAsText: i18n.translate(
+      'xpack.triggersActionsUI.sections.ruleDetails.eventLogColumn.totalSearchDuration',
+      {
+        defaultMessage: 'Total search duration',
+      }
+    ),
+    isSortable: SORTABLE_COLUMNS.includes('total_search_duration'),
+  },
+  {
+    id: 'es_search_duration',
+    displayAsText: i18n.translate(
+      'xpack.triggersActionsUI.sections.ruleDetails.eventLogColumn.esSearchDuration',
+      {
+        defaultMessage: 'ES search duration',
+      }
+    ),
+    isSortable: SORTABLE_COLUMNS.includes('es_search_duration'),
+  },
+  {
+    id: 'schedule_delay',
+    displayAsText: i18n.translate(
+      'xpack.triggersActionsUI.sections.ruleDetails.eventLogColumn.scheduleDelay',
+      {
+        defaultMessage: 'Schedule delay',
+      }
+    ),
+    isSortable: SORTABLE_COLUMNS.includes('schedule_delay'),
+  },
+  {
+    id: 'timed_out',
+    displayAsText: i18n.translate(
+      'xpack.triggersActionsUI.sections.ruleDetails.eventLogColumn.timedOut',
+      {
+        defaultMessage: 'Timed out',
+      }
+    ),
+    isSortable: SORTABLE_COLUMNS.includes('timed_out'),
+  },
+];
+
+const API_FAILED_MESSAGE = i18n.translate(
+  'xpack.triggersActionsUI.sections.ruleDetails.eventLogColumn.apiError',
+  {
+    defaultMessage: 'Failed to fetch execution history',
+  }
+);
+
+const RULE_EVENT_LOG_LIST_STORAGE_KEY = 'xpack.triggersActionsUI.ruleEventLogList.initialColumns';
+
+export const DEFAULT_INITIAL_VISIBLE_COLUMNS = [
```

Comment:
This looks like a good list of columns to start with!  Once this is live, we'll probably have a better sense if there's more we need.